src/core/inline-idl-parser.js

```diff
@@ -19,45 +19,47 @@ const methodSplitRegex = /\.?(\w+\(.*\)$)/;
 function parseInlineIDL(str) {
   const [nonMethodPart, methodPart] = str.split(methodSplitRegex);
   const tokens = nonMethodPart
-    .split(".")
+    .split(/[./]/)
     .concat(methodPart)
-    .filter(s => s && s.trim());
+    .filter(s => s && s.trim())
```

Comment:
maybe should use `array.reduce` here?

Reply:
🤔 what’s there looks ok.

Reply:
didn't like `s.trim()` being called twice

Reply:
Ah, I see. They are semantically slightly different because of the usage context. The filter could regex to .test() matching just whitespace... but trim() does declaratively the same thing.